### DISCUSSION TITLE
self-hosting/get-started

### DISCUSSION THREAD

**Discussion by giscus** [graphql-hive/console#1760] [Docs Discussions]
# self-hosting/get-started

GraphQL Hive Documentation

https://docs.graphql-hive.com/self-hosting/get-started

**Comment by Bjohnson131** [comment]
Hello, the `wget` command has fallen due to link rot. Here is the updated command as of 3/17/23: `wget https://raw.githubusercontent.com/kamilkisiela/graphql-hive/main/docker/docker-compose.community.yml`
